themes/beaver/layouts/partials/page/analytics.html

```
@@ -1,3 +1,15 @@
+{{ if and (not hugo.IsServer) (site.Params.microsoftClarity) }}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)